Lesson24/selcan1/index..html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.2/css/all.min.css" integrity="sha512-SnH5WK+bZxgPHs44uWIX+LLJAJ9/2PkPKZ5QiAj6Ta86w+fsb2TkcmfRyVX3pBnMFcV7oQPJkl9QevSCWr3W6A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Document</title>
+    <style>
+        .kapsayici{
+            width: 100%;;
+            height:100vh;
+            background-color: rgb(60, 60, 104); 
+            display: flex; 
+            justify-content: center;  
+            align-items: center;   
+          }
+        .anaMenu{
+            border-radius: 25px;
+            background-color: white;
+            width: 50%;
```

Comment:
Don't specify width and height like this. In particular, in cases like here, here specify explicitly both width and height as your content will probably overflow on some device.

Instead, you can specify 'width' and 'max-width' property (.e.g. `width: 100%; max-width: 1000px;`) and don't set height at all so that the content takes as much space as it needs on smaller screens.